test/EndToEnd/NuGet.Tests.psd1

```diff
@@ -0,0 +1,85 @@
+@{
+
+# Script module or binary module file associated with this manifest
+ModuleToProcess = 'NuGet.Tests.psm1'
+
+# Version number of this module.
+ModuleVersion = '1.0.0.0'
+
+# ID used to uniquely identify this module
+GUID = 'de6372a1-c5d9-4a64-8321-e6f8121587e2'
+
+# Author of this module
+Author = 'NuGet Team'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
```

Comment:
Copied this manifest from the product module [NuGet.psd1](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGetConsole.Host.PowerShell/Scripts/NuGet.psd1). Also as the test module is not to be distributed with the VSIX it doesn't really matter.